Microsoft.Azure.Cosmos/src/CosmosClientOptions.cs

```diff
@@ -346,6 +346,23 @@ public ConnectionMode ConnectionMode
         /// <seealso cref="TransactionalBatchItemRequestOptions.EnableContentResponseOnWrite"/>
         public bool? EnableContentResponseOnWrite { get; set; }
 
+        /// <summary>
+        /// Gets or sets the advanced replica selection flag. The advanced replica selection logic keeps track of the replica connection
+        /// status, and based on status, it prioritizes the replicas which are connected to the backend, so that the requests can be sent
+        /// confidently to the particular replica. This helps the cosmos client to become more resilient and effictive to any connection
```

Comment:
```suggestion
        /// This helps the cosmos client to become more resilient and effective to any connectivity issue.
```

Some wording suggestions

Reply:
Updated the verbiage.

Reply:
Per our conversation, public API contract changes doesn't apply any more.